mailer.js

```diff
@@ -11,6 +12,24 @@ var DEFAULT_LOCALE = 'en'
 var DEFAULT_TIMEZONE = 'Etc/UTC'
 
 module.exports = function (log) {
+  // Email template to UTM campaign map
+  var templateNameToCampaignMap = {
+    'passwordResetRequiredEmail': 'password-reset-required',
+    'passwordChangedEmail': 'password-changed-success',
```

Comment:
How come `password-changed-success` has the `-success` suffix, but `password-reset` does not?